tests/run/libcpp0x_all.pyx

```diff
@@ -0,0 +1,17 @@
+# distutils: extra_compile_args=-std=c++0x
+# tag: cpp
```

Comment:
Instead of adding explicit CFLAGS here, there should be a tag `cpp11` to also exclude the test if C++11 is not supported.